src/server_manager/web_app/ui_components/outline-intro-step.js

```diff
@@ -276,20 +278,24 @@ Polymer({
     },
     isDigitalOceanAccountConnected: {
```

Comment:
Delete this property and call the function in the template instead

Reply:
Done